phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainersSidebar.java

```diff
@@ -104,14 +104,17 @@ private ContainersSidebarToggleGroup createSidebarToggleGroup() {
     private ListWidgetSelector createListWidgetSelector(CombinedListWidget<ContainerDTO> availableContainers) {
         ListWidgetSelector listWidgetSelector = new ListWidgetSelector();
 
-        listWidgetSelector.setSelected(javaFxSettingsManager.getContainersListType());
-        listWidgetSelector.setOnSelect(type -> {
-            availableContainers.showList(type);
+        availableContainers.selectedListWidgetProperty().bind(listWidgetSelector.selectedProperty());
 
-            javaFxSettingsManager.setContainersListType(type);
-            javaFxSettingsManager.save();
+        listWidgetSelector.selectedProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
With which other code should this code be unified?
The selection settings of each sidebar are saved separately, via a different method (`setContainersListType` for the containers, `setAppsListType` for the applications, etc.).
I don't know how we could unify this.

Additionally, I'm already planning to refactor the sidebar classes with MVC, but it will still take a few days until I can start with that.

Reply:
Ok.